README.md

```diff
@@ -763,6 +763,7 @@ _Command-line file upload_
 - [MapBBCodeShare](https://github.com/MapBBCode/share.mapbbcode.org) - Tool for sharing custom OSM maps. Support for annotated markers, polygons, lines, multi-format import/export, multiple layers, shortlinks. ([Demo](http://share.mapbbcode.org/)) `WTFPL/Other` `PHP`
 - [OpenGTS](http://www.opengts.org/) - Entry-level fleet tracking system. Supports variety of tracking devices and protocols. Comes with rich web-interface and reporting features. ([Demo](http://track.opengts.org/track/Track), [Source Code](https://sourceforge.net/projects/opengts/files/server-base/)) `Apache-2.0` `Java`
 - [Openstreetmap](http://www.openstreetmap.org/) - OpenStreetMap is a map of the world, created by people like you and free to use under an open license. ([Source Code](https://github.com/openstreetmap/openstreetmap-website)) `GPL-2.0` `Ruby`
+- [Orion](https://github.com/LINKIWI/orion-web) - Powerful OwnTracks API-compliant location data visualization frontend for the web. ([Demo](https://linkiwi.github.io/orion-web/)) `MIT` `Javascript`
```

Comment:
> You'll need Node v8 or later, any recent version of npm

Please replace Javascript with Nodejs

Reply:
The linked repository is a frontend application, and not written in Node. But I can change it to Node anyway if that is preferred.

Reply:
Is installing https://github.com/LINKIWI/orion-server required for https://linkiwi.github.io/orion-web to work? Does a working setup require a Node stack on the server? If yes, please change the language tag. If there is an installation method that does not require server-side JS (only serving JS to the client), `Javascript` is fine.

Reply:
> Is installing https://github.com/LINKIWI/orion-server required for https://linkiwi.github.io/orion-web to work?

Yes, the server component is required. There doesn't seem to be a convention yet for listing projects that are distributed across multiple repositories, so this entry only links the frontend component. The server is written in Python, while the frontend is written in JS (but requires Node for installation).

> Does a working setup require a Node stack on the server? If yes, please change the language tag.

Yes, and I've pushed a new commit that changes the language to Node.

Thanks

Reply:
I should clarify that the frontend and backend components are entirely independent and agnostic of each other. You could run one without the other and nothing would break; it just wouldn't be very useful.

Reply:
> Yes, the server component is required
> The server is written in Python, while the frontend [...] requires Node for installation

That would be `Python/Nodejs` then. <-- It helps evaluating the general requirements by simply reading the list entry.

> There doesn't seem to be a convention yet for listing projects that are distributed across multiple repositories

This is fine, you can link to either one. Better make sure your project READMEs have links to each other (currently they don't) because that's what confused me. I didn't know there was a server component until you told me about it. Thanks